pyanaconda/ui/gui/spokes/root_password.py

```diff
@@ -79,8 +77,10 @@ def initialize(self):
         self._password_confirmation_entry = self.builder.get_object("password_confirmation_entry")
         self._password_bar = self.builder.get_object("password_bar")
         self._password_label = self.builder.get_object("password_label")
-        self._lock = self.builder.get_object("lock")
+        self._enable_radio = self.builder.get_object("enable_radio")
+        self._disable_radio = self.builder.get_object("disable_radio")
```

Comment:
I would like to change these variable names. I'm missing the target here. What to enable root account, ssh access, password field?

Reply:
I added "root" in the middle of the names, eg. disable_root_radio.